apps/www/src/content/docs/contribution.md

```diff
@@ -0,0 +1,221 @@
+
+8. **examples** -
+Holds all examples not part of `/docs`, like the [main page](/).
+
+9. **lib/registry** -
+The main folder hosts the source code for different styles of every component. This is likely the main folder you'll be changing.
+
+We support two different styles for every component in `shadcn/vue`:
+
+1. Default
+2. New York
+
+Every component added to the repository must support both versions, including the main source code and associated demos.
+
+## SFC - Single File Components
+
+Multiple components are integrated into one file in `shadcn/ui` - the React version of `shadcn` - while Vue only supports one component per file, hence the name Single File Component (SFC). In such cases, you need to create separate files for each component part and then export them all in an `index.ts` file.
+
+See the [`Accordion`](https://github.com/radix-vue/shadcn-vue/tree/v0.10.2/apps/www/src/lib/registry/default/ui/accordion) source code as an example.
+
+## Wrapping Radix-Vue Components
+
+[Radix-Vue](https://www.radix-vue.com) hosts many low-level UI components that are used to build reusable components.
+There are many cases that you need to wrap `Radix-Vue` components.
+
+### Props & Events
+All of the `Radix-Vue` compoennts expose their prop and emit types. We need to forward any props/events that are coming from outside to the `Radix-Vue` component.
+
+To do so, we have a helper function named [`useForwardPropsEmits`](https://www.radix-vue.com/utilities/use-forward-props-emits.html) that combines props and events that must be binded to the child radix component.
+
+To be more clear, the function `useForwardPropsEmits` takes in props and an optional emit function, and returns a
+computed object that combines the parsed props and emits as props.
+
+Here's an example from `Accordian` root component.
+
+```vue
+<script setup lang="ts">
+import {
+  AccordionRoot,
+  type AccordionRootEmits,
+  type AccordionRootProps,
+  useForwardPropsEmits,
+} from 'radix-vue'
+
+const props = defineProps<AccordionRootProps>()
+const emits = defineEmits<AccordionRootEmits>()
+
+const forwarded = useForwardPropsEmits(props, emits)
+</script>
+
+<template>
+  <AccordionRoot v-bind="forwarded">
+    <slot />
+  </AccordionRoot>
+</template>
+```
+
+As you can see, `AccordionRootEmits` and `AccordionRootProps` types are imported from radix, combined with `useForwardPropsEmits` and then are binded using `v-bind` syntaxt.
+
+### CSS Classes
+There are cases when we want to accept `class` as a prop in our `shadcn/vue` components and then combine it with a default style on our `radix-vue` component.
+
+In these cases, we can not use `v-bind` while `class` is declared as a prop, because this would lead in [double class binding](https://github.com/radix-vue/shadcn-vue/pull/241), so the `class` propert must be dealt specefically.
```

Comment:
We can use `v-bind` when class is declared as prop 

we only use this way to prevent class duplication with `cn` function

Reply:
Yup. As long as we have `class` as prop, we can use `v-bind` 😁

Reply:
Thanks for the feedback ! I tried to be more clear on this one.

Reply:
I also added some additional information based on `CONTRIBUTING.md`.